src/python/grpcio/tests/interop/methods.py

```diff
@@ -305,6 +307,30 @@ def _oauth2_auth_token(stub, args):
         'expected to find oauth scope "%s" in received "%s"' %
             (response.oauth_scope, args.oauth_scope))
 
+def _jwt_token_creds(stub, args):
```

Comment:
Obviously, this flaw is slightly unrelated, but I think it would be good to fix this anyway.